pytissueoptics/rayscattering/tissues/tissue.py

```diff
@@ -0,0 +1,24 @@
+from typing import List
+
+from pytissueoptics.scene import MayaviViewer, Material
+from pytissueoptics.scene.solids import Solid
+
+
+class Tissue:
+    def __init__(self, solids: List[Solid]):
+        self._solids = solids
+
+    def getSolids(self) -> List[Solid]:
+        return self._solids
+
+    def addToViewer(self, viewer: MayaviViewer):
```

Comment:
Should we really inverse dependency on MayaviViewer?

Reply:
Je comprends que ça aide avec le display pour le user si les params et les methodes pour display sont déjà préfaites pour lui. Let me get warmed up.

Reply:
dans tous les cas la dépendance doit toujours être vers le viewer.
Après le call est différent en effet. On pourrait peut-être se faire une interface de Scene pour le viewer mais ce serait funky. Je sais pas trop comment améliorer ca. Ca fait juste isoler la poutine de configuration du display, mais elles ne sont pas préfaites pour lutilisateur s'il se fait sa propre scene/tissue. 

Par exemple sur SensorSim, jai le même call et je configure manuellement le display de chaque solide:
![image](https://user-images.githubusercontent.com/29587649/160848309-01c41b54-8e7d-4d0d-b1e8-7626a6d06ceb.png)

Reply:
on pourrait ajouter une implémentation par défaut au moins dans notre interface Tissue. qui va essayer de tout display en gris genre

Reply:
accepte apres discussion